src/Tizen.NUI/src/public/WebView/WebSettings.cs

```diff
@@ -113,7 +113,7 @@ public bool WebSecurityEnabled
         /// <summary>
         /// Enable cache builer
```

Comment:
typo

Reply:
Done.
I updated it in another patch https://github.com/Samsung/TizenFX/pull/3520